app/uk/gov/hmrc/individualsemploymentsapi/service/v2/ScopesHelper.scala

```diff
@@ -25,6 +25,33 @@ import play.api.libs.json.JsValue
 
 class ScopesHelper @Inject()(scopesService: ScopesService) {
 
+  /**
+    * @param scopes The list of scopes associated with the user
+    * @param endpoints The endpoints for which to construct the query string
+    * @param payeRef The PAYE reference
+    * @return A google fields-style query string with the fields determined by the provided endpoint(s) and scopes
+    */
+  def getQueryStringWithParameterisedFilters(scopes: Iterable[String],
+                                             endpoints: List[String],
+                                             employerRef: String): String = {
+    val queryString = getQueryStringFor(scopes, endpoints)
+
```

Comment:
Possibly remove the space here? Picky :-)

Reply:
Or do this on a single line 

getQueryStringFor(scopes, endpoints).replace("<employerRef>", employerRef)